src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -263,10 +266,12 @@ private struct InterpolatedStringScanner
             public SyntaxDiagnosticInfo? Error = null;
             private bool EncounteredUnrecoverableError = false;
 
-            public InterpolatedStringScanner(Lexer lexer, bool isVerbatim)
+            public InterpolatedStringScanner(Lexer lexer)
             {
                 _lexer = lexer;
-                _isVerbatim = isVerbatim;
+
+                _isVerbatim = (lexer.TextWindow.PeekChar(0) == '$' && lexer.TextWindow.PeekChar(1) == '@') ||
+                              (lexer.TextWindow.PeekChar(0) == '@' && lexer.TextWindow.PeekChar(1) == '$');
```

Comment:
only place where this is computed now.  note: this will get much more complex in the raw-string-literal case in the fuure.  having it be cenralized means only one place to update.